### DISCUSSION TITLE
Manually add students returns "No students created."

### DISCUSSION THREAD

**Discussion by MarcoMobilio** [community/community#17739] [GitHub Education]
Hi everyone,

I started using GitHub Classroom this year, but I am facing an issue.
I successfully created the roster for my class (183 students), however if I now tray to manually add a student by selecting "update students" and then adding a new student manually it returns to the list of students with just a message "no students created." and no error or anything else... Am I missing something stupid?

**Comment by octosteve** [comment]
That's usually the case when the student is already in the list. Is there any chance they're already on your roster?

**Reply by MarcoMobilio** [reply]
Unfortunately, that is not the case, It just seems to be unable to modify the roster after the initial creation. It always says that no students were added. I think I saw other, newer, comments related to the same issue.
